src/main/js/components/subcomponents/Order.js

```diff
@@ -2,4 +2,256 @@
+                            <i className="fas fa-sync-alt" style={{fontSize: '20px'}}>    Update Cart</i>
+                        </button>
+
+                        <span className="cart-total name">Cart Totals: </span>
+                        <span className="cart-total amount">{this.props.location.state.diningSession.price}</span>
+                    </div>
+                    <footer>
+                        <div className="signature">
+                            <h6>Sushi</h6>
+                            <h5>PotatoPeeps</h5>
+                        </div>
+                    </footer>    			
+                </main>
+                
+            </div>
+        );
+    }
+
+    // this method dynamically loads the orders associated with the given
+    // diningSession and converts them into renderable components, namely StaffOrderList and StaffOrder
+    requestOrders(){
+        fetch(this.props.location.diningSession.entity._links.orders.href, {method: 'GET', headers: {'Content-Type': 'application/json'}})
+           .then(
+               response => {
+                   if (!response.ok) {
+                       console.log('Looks like there was a problem. Status Code: ' +
+                           response.status);
+                       return;
+                   }
+
+                   // Examine the text in the response
+                   response.json().then((data) => {
+                       console.log(data._embedded.orders);
+                       this.setState(
+                           {
+                                orders : data._embedded.orders
+                            }
+                        );
+                        console.log("In the render of Orders");
+                        console.log(this.state);
+                        console.log(this.state.orders);
+                        console.log("in render");
+                   });
+               }
+           )
+           .catch(function(err) {
+               console.log('Fetch Error :-S', err);
+           });
+   }
+
+
+   componentDidMount(){
+        this.requestOrders();
+   }
+
+   componentWillUnmount(){
+       console.log("UNMOUNTING UNMOUNTING UNMOUNTING UNMOUNTING UNMOUNTING")
+   }
+
+   handleBackClick(){
+    // this.props.history.goBack();
+    this.props.history.push
+        (
+            {
+                pathname: ('/staff/'),
+                props: this.props
+            } 
+        );
+   }
+}
+
+class StaffOrderList extends React.Component{
+    constructor(props){
+        super(props);
+        
+    }
+
+    render(){
+        const orders = this.props.orders.map(order =>
+            <StaffOrder 
+                key={order._links.self.href}
+                order={order}
+                orderAttributes={this.props.orderAttributes}
+                onUpdate={this.props.onUpdate}
+                //TODO does Staff have authority to delete order?
```

Comment:
Not included in the stories we provided. As such it is not functionality we should implement